client/src/app/site/deployment-center/deployment-center-setup/wizard-logic/azure-devops.service.ts

```diff
@@ -97,7 +97,10 @@ export class AzureDevOpsService implements OnDestroy {
     });
   }
 
-  startDeployment(account: string, deploymentObj: any) {
+  startDeployment(account: string, deploymentObj: any, isNewVsoAccount: boolean) {
+    if (isNewVsoAccount) {
+      this._accountsList = [];
```

Comment:
Should we instead clear/refresh the account list at the point where we create the new account? That way other references to getAccounts() will also get benefitted?

Reply:
We are clearing accountsList right after new account creation only, have a look at the function _startVstsDeployment().
Its just that we are not creating explicit function to clear the accountsList since no other reference is there for getAccounts() after creating new account and before clearing (newly added code) accountsList.

Reply:
Even though functionally they are the sam, I'd recommend to clear/refresh in _startVstsDeployment() method of deployment-center-state-manager.ts itself. - where the org gets created.

Reply:
I agree with Amal that it might be a better choice to clear after account creation in _startvsts.. method

Reply:
as discussed, _accountsList is a private variable in different class, so this will require a major change and this doesn't seems much of a need in current scenario.